test/fuzz/transformation_add_type_float_test.cpp

```diff
@@ -70,6 +70,17 @@ TEST(TransformationAddTypeFloatTest, IsApplicable) {
   transformation = TransformationAddTypeFloat(7, 32);
   ASSERT_TRUE(
       transformation.IsApplicable(context.get(), transformation_context));
+
+  // By default, SPIR-V does not support 64-bit float types.
+  // Below we add such capability, so the test should now pass.
+  context.get()->get_feature_mgr()->AddCapability(SpvCapabilityFloat64);
+  ASSERT_TRUE(TransformationAddTypeFloat(7, 64).IsApplicable(
+      context.get(), transformation_context));
+
+  // Should not be able to add float type of width different from 16/32/64
+  ASSERT_DEATH(TransformationAddTypeFloat(7, 20).IsApplicable(
+                   context.get(), transformation_context),
+               "Unexpected float type width");
 }
```

Comment:
I like this! But note that death tests need to be surrounded by:

```
#ifndef NDEBUG
  ASSERT_DEATH(...);
#endif
```

The reason is that death tests cannot be present in release mode builds (because assertions get preprocessed away).